app/views/aboutyou/TaxAvoidanceSchemesAgentView.scala.html

```diff
@@ -0,0 +1,81 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.aboutyou.TaxAvoidanceSchemes
+@import viewmodels.LegendSize
+@import components.ParagraphBody
+
+@this(
+    layout: templates.Layout,
+    formHelper: FormWithCSRF,
+    govukErrorSummary: GovukErrorSummary,
+    govukCheckboxes: GovukCheckboxes,
+    govukButton: GovukButton,
+    govukDetails: GovukDetails,
+    p: ParagraphBody,
+    config: FrontendAppConfig
+)
+
+@(form: Form[_], mode: Mode, taxYear: Int)(implicit request: Request[_], messages: Messages)
+
+@details = {
+    <p>
+        @messages("taxAvoidanceSchemes.understandingAvoidance.summary.p")
+        <a target="_blank" href="@{config.understandingTaxAvoidanceUrl}">@messages("taxAvoidanceSchemes.understandingAvoidance.summary.findOutMore")</a>
+    </p>
+}
+
+@details1 = {
+    <p>
+        @messages("taxAvoidanceSchemes.understandingRemuneration.summary.p")
+        <a target="_blank" href="@{config.understandingRemunerationUrl}">@messages("taxAvoidanceSchemes.understandingRemuneration.summary.findOutMore")</a>
+    </p>
+}
+
+@layout(pageTitle = title(form, messages("taxAvoidanceSchemes.agent.title")), taxYear = taxYear) {
+
+    @formHelper(action = controllers.aboutyou.routes.TaxAvoidanceSchemesController.onSubmit(mode, taxYear), Symbol("autoComplete") -> "off") {
+
+        @if(form.errors.nonEmpty) {
+            @govukErrorSummary(ErrorSummaryViewModel(form, errorLinkOverrides = Map("value" -> "value_0")))
+        }
+
+        <h1 class="govuk-heading-l">@messages("taxAvoidanceSchemes.agent.heading")</h1>
```

Comment:
can't we use component for this?

Reply:
I need the link as part of the paragraph which the component does not allow unless I am wrong?